<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="01hw2164715" failures="0" tests="1" name="com.perfecto.runner.webRunner" time="148.632" errors="1" timestamp="20 Dec 2023 13:29:50 GMT">
  <testcase classname="com.perfecto.runner.webRunner" name="runScenario" time="148.632">
    <error message="One or more fields not found" type="java.lang.Exception">
        <![CDATA[java.lang.Exception: One or more fields not found
	at com.perfecto.pageObjects.FACapturaDatosAsistenciaComiteCTC.SetValuesToFields(FACapturaDatosAsistenciaComiteCTC.java:163)
	at com.perfecto.stepdefinition.FACapturaDatosAsistenciaComiteCTCSteps.set_values_to_fields(FACapturaDatosAsistenciaComiteCTCSteps.java:36)
	at âœ½.set values to fields 'Arjona', 'Moreno' and 'Venavides'(file:///C:/Users/2357672/Downloads/FWB_Perfecto_v9/FWB-Perfecto/src/main/resources/Features/SICRED_Flow.feature:255)
]]>
    </error>
  </testcase> <!-- runScenario -->
</testsuite> <!-- com.perfecto.runner.webRunner -->
